### DISCUSSION TITLE
続きのページへのヘッダー？

### DISCUSSION THREAD

**Discussion by jasondee1992** [kmuto/review#1844] [Q&A] [answered]
継続ページのヘッダーとヘッダーアンダースコアを削除するにはどうすればよいですか？

継続段落がありますが、次のページには常にundelineヘッダーが特別に表示されるヘッダーがあります。

これはPREDEFとCHAPTERに表示されています

![image](https://user-images.githubusercontent.com/49851974/181653282-78391f46-e29d-43f4-878c-824bf9c50636.png)

**Comment by kmuto** [comment, marked as answer]
review-jsbook版なので、
sty/review-custom.styに以下を追加すればおそらく消えるでしょう。

```
\fancyhead{}
\renewcommand{\headrulewidth}{0pt}
```

なお、提示されているサンプルでは紙面サイズを超えているようです。
config.ymlのtexdocumentclassで「media=print」としている場合、これは印刷所で印刷することを前提にしたPDFを出力します。
印刷所での印刷依頼のために、実際の印刷の紙のサイズの四隅にマークを入れています(トンボといいます)。つまり、実際に印刷製本される紙のサイズはこのマークの内側です。
おそらくjasondee1992さんの目的では、media=printをmedia=ebook (電子版用に実際の紙サイズだけにする)に変えたほうがよいでしょう。各マージンサイズ/行数などについても再設定してください。

**Reply by jasondee1992** [reply]
良い！ kmutoさん、どうもありがとうございました。
